app/controllers/wiki_controller.rb

```diff
@@ -36,6 +36,18 @@ def show
       end
     end
 
+    if @node && @node.has_power_tag('abtest')
+      if current_user == nil || (current_user.role != 'admin' && current_user.role != 'moderator')
+        if Random.rand(2) == 0
+          redirect_to DrupalNode.find(@node.power_tag('redirect')).path
+        end
+        return
+      elsif (current_user.role == 'admin' || current_user.role == 'moderator')
+        flash.now[:warning] = "Only moderators and admins see this page, as it is redirected to #{DrupalNode.find(@node.power_tag('abtest')).title}.
```

Comment:
Ah, I meant in the flashed message. Because it'll be confusing to folks if it only happens sometimes. Thanks!